test/functional/connection.test.js

```diff
@@ -70,6 +71,21 @@ describe('Connection - functional', function () {
     }
   });
 
+  it('topology "open" event should only pass topology (and not error)', function (done) {
```

Comment:
Usually the test description reads like a sentence (ex. "it should correctly create X" or "it should throw an error if Y happens", you can see more examples within this connection.test file). I would try to stay consistent with that wording. Maybe you could say something like `'should only pass one topology argument for topology "open" events'`, or you could create another `describe` block if you think what you're testing should fall under a different category than the `Connection - functional` descriptor.